src/common-declarations.js

```diff
@@ -53,8 +53,19 @@ const commonDeclarations = {
   paddingBottom: [ 0 ],
   paddingLeft: [ 0 ],
 
+  minWidth: [ '100%' ],
   maxWidth: [ '100%' ],
-  height: [ 'auto' ],
+  width: [
+    'auto',
+    0,
+    '50%',
```

Comment:
Yep, that's definitely subjective, and it'd be easy to start going down the rabbit hole with 25%, 75%, 10%, 20%, 30%, etc. I'll just remove that one. Thanks :)